catalog/app/containers/Admin/Status/gql/Status.graphql

```diff
@@ -0,0 +1,24 @@
+query ($window: Int) {
```

Comment:
It's confusing to see the name "window" in the frontend environment meaning something different from UI windows. Maybe, "timespan" or "time frame" fits better?

Reply:
As I see, we use it only for ANALYICS_WINDOW constant. It think, it's better to rename it, because the naming is confusing

Reply:
there's two more instances of `window` query parameter in our gql schema, and by analogy with ANALYTICS_WINDOW* js variable im introducing a new one -- STATS_WINDOW. i dont feel its confusing when seen in context. we can refactor it later in one take if we come up with a better language.